### ISSUE TITLE
Second Nested Base Network for Cifar10 and Cifar100

### ISSUE THREAD

**Issue by botanyldrm** [yingyichen-cyy/Nested-Co-teaching#3]
I look the instructions given in the repository but could not see how to train second nested network before fine-tuning with co-teaching for Cifar10 and Cifar100. Which strategy should be followed here? Should be generate a second synthetically corrupted Cifar10-Cifar100 dataset for second network training?

**Comment by yingyichen-cyy** [comment]
Dear botanyldrm,
Thank you again for supporting our work!
For the two base nested networks, we only need to follow exactly the same strategy and train on the same datasets.
The two networks will definitely be different since we inject randomness in the networks by applying nested dropout.
You only need to conduct the same command twice as follows:
``` Bash
cd nested/ 
# CIFAR-10 Symmetric 20%, Nested=10
## model 1
python train_resnet.py --train-dir ../data/CIFAR10/train_sn_0.2/ --val-dir ../data/CIFAR10/val/ --dataset CIFAR10 --arch resnet18 --out-dir ./checkpoints/cifar10sn0.2_nested10_model1 --nested 10 --gpu 0

## model 2
python train_resnet.py --train-dir ../data/CIFAR10/train_sn_0.2/ --val-dir ../data/CIFAR10/val/ --dataset CIFAR10 --arch resnet18 --out-dir ./checkpoints/cifar10sn0.2_nested10_model2 --nested 10 --gpu 0
``` 
Note that we generate two balanced training subset for Clothing1M datasets only because for methods such as [DivideMix](https://arxiv.org/abs/2002.07394),  [PLC](https://openreview.net/forum?id=ZPa2SyGcbwh), [ELR+](https://arxiv.org/abs/2007.00151), [MLNT](https://arxiv.org/abs/1812.05214), they sample 1000 mini-batches from the training data for each epoch while ensuring the labels (noisy) are balanced. We only use two fixed balanced subset instead. Sampling balanced subset is very important for having good performance on this Clothing1M benchmark dataset.

**Comment by botanyldrm** [comment]
Thanks a lot for your answer yingyichen-cyy. I will follow your instructions in my works.
